taigit/src/libraries/Taiga.ts

```diff
@@ -9,4 +9,34 @@ taiga_login(username : string, password : string) : Promise<boolean> {
     })
 
     return (response.status == 200);
-}
\ No newline at end of file
+}
+
+//after auth, given slug of a project, will return sprint information.
```

Comment:
It looks like this file is the only one that has changed in the PR. I would keep these changes, but remove the rest.